crates/medmodels-core/src/medrecord/group_mapping.rs

```diff
@@ -506,6 +638,17 @@ mod test {
         assert_eq!(1, group_mapping.groups_of_node(&"0".into()).count());
     }
 
+    #[test]
+    fn test_groups_of_edge() {
```

Comment:
Would we need an invalid_groups_of_edge test or invalid_groups_of_node?
Or is there a case where the count of these groups should be 0?

Reply:
the `invalid_` tests are tests for when an error gets returned. The `groups_of_edge` and `groups_of_node` always return an iterator (which might be empty) so there is no invalid way of calling this function.
Checking for an iterator of size zero checks the exact same code path that is already being checked, so this is not necessary imo.